chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -258,7 +258,8 @@
 
         when os =~ /^(open)?suse/
           append << "install=#{install_url} autoyast=#{node_url}/autoyast.xml"
-          append << "ifcfg=dhcp4 netwait=60"
+          append << " ifcfg=dhcp4 netwait=60"
+          append << " autoupgrade=1" if mnode[:state] == "os_upgrading"
```

Comment:
I don't think you need the leading ` `: we are joining the items of `append` (`append.join(" ")`).